plug.vim

```diff
@@ -1260,7 +1261,10 @@ class Command(object):
   def terminate(self):
     """ Terminate process and cleanup. """
     if self.alive:
-      os.killpg(self.proc.pid, signal.SIGTERM)
+      if not G_IS_WIN:
```

Comment:
I try and avoid negations with if's when I can just invert which block comes first. Reads more naturally. Can you just rearrange this so that it is

````python
if G_IS_WIN:
  os.kill(self.proc.pid, signal.SIGINT)
else:
  os.killpg(self.proc.pid, signal.SIGTERM)
```